dace/config_schema.yml

```diff
@@ -168,25 +168,38 @@ required:
                         default: ''
                         
             #############################################
-            # GPU (CUDA) compiler
+            # GPU (CUDA/HIP) compiler
             cuda:
                 type: dict
                 title: GPU
-                description: GPU (CUDA) compiler preferences
+                description: GPU (CUDA/HIP) compiler preferences
                 required:
+                    backend:
+                        type: str
+                        default: 'cuda'
+                        title: Compilation backend
+                        description: >
+                            Backend to compile for ('cuda' for NVIDIA or 'hip' for AMD).
+
                     path:
                         type: str
                         default: ''
-                        title: CUDA path override
-                        description: Path to CUDA toolkit root directory
+                        title: CUDA/HIP path override
+                        description: Path to CUDA toolkit or ROCm/HIP root directory
 
                     args:
                         type: str
-                        title: Arguments
-                        description: Compiler argument flags
+                        title: nvcc Arguments
+                        description: Compiler argument flags for CUDA
                         default: '-std=c++14 -Xcompiler -fPIC -O3 -Xcompiler -march=native --use_fast_math -Xcompiler -Wno-unused-parameter'
                         default_Windows: '-std=c++14 -O3 --use_fast_math'
 
+                    hip_args:
```

Comment:
Hmm, I know we will reorganize the codegens further in the future, but is it a good idea to also fix these option paths to have HIP under CUDA?

Reply:
The name will change to GPU. Once we do, do you prefer `compiler.hip.args` or `compiler.gpu.hip_args`? Some of the options are shared between the two backends.

Reply:
I don't think there's any advantage in sharing arguments, it's unlikely that anyone will develop for both simultaneously